lib/ddtrace/ext/runtime.rb

```diff
@@ -7,6 +7,7 @@ module Runtime
       LANG = 'ruby'.freeze
       LANG_INTERPRETER = (RUBY_ENGINE + '-' + RUBY_PLATFORM).freeze
       LANG_VERSION = RUBY_VERSION
+      RUBY_ENGINE =  ::RUBY_ENGINE # e.g. 'ruby', 'jruby', 'truffleruby'
```

Comment:
will this blowup anything in the frontend UI that relies on the `ruby` tag?

Reply:
Not sure I follow, `RUBY_ENGINE` is being used to decide which implementation to use by default here: https://github.com/DataDog/dd-trace-rb/pull/1172/files#diff-6650ecd3c000f2a31d4f5335841789a8R213-R217

That it's only use, and it's never exposed anywhere in the tracer.

Reply:
derp, i got mixed up with `LANG`...disregard 🤦